vhdbuilder/packer/buildperformance/main.go

```diff
@@ -0,0 +1,62 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/Azure/agentBaker/vhdbuilder/packer/build-performance/pkg/service"
+)
+
+func main() {
+	// Recover from panic and exit gracefully in order to prevent failing pipeline step
+	defer func() {
+		if r := recover(); r != nil {
+			fmt.Println("Recovered from panic in main:", r)
+			os.Exit(0)
+		}
+	}()
+
+	config, err := service.SetupConfig()
+	if err != nil {
+		panic(err)
+	}
+	fmt.Println("Program config set")
+
+	maps := service.CreateDataMaps()
+
+	client, err := service.CreateKustoClient(config.KustoEndpoint, config.KustoClientId)
+	if err != nil {
+		panic(err)
+	}
+	defer client.Close()
+	fmt.Println("Kusto client created")
+
+	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Minute)
+	defer cancel()
+
+	if config.SourceBranch == "refs/heads/zb/regression2" {
```

Comment:
im assuming we'd get rid of this once we're ready for merge?

Reply:
Yes, this is just a placeholder to enable functionality for this branch. Once its ready to merge, it will be changed to master. That way master will be the source of truth for data analysis, and other branches will evaluate their performance relative to it.